### PR TITLE
feat(`vm`): add keyExists

### PR DESCRIPTION
**Pull request by Evalir**
Upstream from https://github.com/foundry-rs/foundry/pull/5431

### CODE REVIEW

File: src/Vm.sol [hunk 1]
```diff
@@ -259,6 +259,9 @@ interface VmSafe {
     function parseJsonBytes32(string calldata, string calldata) external returns (bytes32);
     function parseJsonBytes32Array(string calldata, string calldata) external returns (bytes32[] memory);
 
+    // Checks if a key exists in a JSON or TOML object.
+    function keyExists(string calldata, string calldata) external view returns (bool);
```
Comment (mds1) [inline src/Vm.sol:263]: We try to always used named inputs/outputs here for clarity, do you mind updating? (looks like the above `parseJson*` methods are also missing named params if you wanted to fix those also 😅

Comment (Evalir) [reply src/Vm.sol:263]: oh yeah just followed what was above 😅 will update them all!

Comment (mds1) [reply src/Vm.sol:263]: hey @Evalir just bumping this

**Comment by Evalir** [comment]
@mds1 should be g2g!

**Review by mds1 — APPROVED** [review]
Thanks!

**Pull request merged**